### PR TITLE
update list

### PR DESCRIPTION
**Pull request by TalaatMagdy**
Before raising a PR and it getting merged, make sure you have completed the following:

### Things to do:

- [ ] Add podcasts in appropriate categories in ascending order of podcast name
- [ ] If there is no matching category, add a new category to the list in ascending order of category name
- [ ] Add a brief description of the podcast
- [ ] Add the frequency of episodes (Check the list for examples)

Thanks for your contributions and being awesome :) Cheers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,10 +177,17 @@ List of podcasts which are helpful for software engineers/programmers.
...
 ## Kotlin
+
 - [Talking kotlin](http://talkingkotlin.com)
 
   - <b>Description</b>: Kotlin and more
   - <b>Frequency</b>  : Approximately once a month
+  
+## Swift
+
+- Swift Unwrapped ([By Spec](https://spec.fm/podcasts/swift-unwrapped) | [iTunes](https://itunes.apple.com/us/podcast/swift-unwrapped/id1209817203?mt=2)
```
Comment (rShetty) [inline README.md]: Can we fix the closing angular bracket here, plz check the preview

File: README.md [hunk 2]
```diff
@@ -194,6 +213,9 @@ List of podcasts which are helpful for software engineers/programmers.
 
 
 ## Web development
+- Does Not Compute ([By Spec](https://spec.fm/podcasts/does-not-compute) | [iTunes](https://itunes.apple.com/us/podcast/does-not-compute/id1048731980?mt=2))
```
Comment (rShetty) [inline README.md]: Can we add a space after the header ?

File: README.md (hunk 1 above)
Comment (TalaatMagdy) [reply README.md]: edit this Thanks!

File: README.md (hunk 2 above)
Comment (TalaatMagdy) [reply README.md]: OK

**Pull request merged**